test/cpu_test.cpp

```diff
@@ -1052,7 +1052,7 @@ TEST(cpu_test, thread_safe_calls_convolution_2d_2items)
     unset_environment("NGRAPH_CPU_CONCURRENCY");
 }
 
-TEST(cpu_test, constant_convertlayout)
+TEST(cpu_test, MLIR_DISABLE_TEST(constant_convertlayout))
```

Comment:
Add a comment as to why it is disabled and what's needed to enable it.

Reply:
done.